definitions/CMakeLists.txt

```diff
@@ -15,13 +15,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.15)
 
 project(Definitions)
 
 find_package(Thunder)
 
-project_version(1.0.0)
+project_version(${Thunder_VERSION})
```

Comment:
Why would this be dependend on a Thunder version? There is no need to align this with Thunder, it is a version on its own, if it was bound to a version, it would be the version of the tools (at least be dependend on a minimum version of those tools. So leave this a "real" version number.